samples/Daslang/fib.das

```diff
@@ -0,0 +1,19 @@
+// from: https://daslang.io/#look
+
+def fibR(n) {
+    if (n < 2) {
+        return n;
+    } else {
+        return fibR(n - 1) + fibR(n - 2);
+    }
+}
+def fibI(n) {
+    var last = 0;
+    var cur = 1;
+    for i in range(0, n-1); {
+        let tmp = cur;
+        cur += last;
+        last = tmp;
+    }
+    return cur;
+}
```

Comment:
This is not a real-world example and it's also incredibly small so not a very good sample representative of the language from which we can train the classifier. Please replace this with 2-3 real world representative samples.